README.md

```diff
@@ -69,110 +90,96 @@ pip install -i https://test.pypi.org/simple/ taichi-nightly
 
-Taichi's documentation is available at https://docs.taichi.graphics.
+### Lectures & talks
 
-# Contacts
+- SIGGRAPH 2020 course on Taichi basics: [YouTube](https://youtu.be/Y0-76n3aZFA), [Bilibili](https://www.bilibili.com/video/BV1kA411n7jk/), [slides (pdf)](https://yuanming.taichi.graphics/publication/2020-taichi-tutorial/taichi-tutorial.pdf).
+- Chinagraph 2020 用太极编写物理引擎: [哔哩哔哩](https://www.bilibili.com/video/BV1gA411j7H5)
+- GAMES 201 高级物理引擎实战指南 2020: [课件](https://github.com/taichi-dev/games201)
+- 太极图形课第一季：[课件](https://github.com/taichiCourse01)
+- [TaichiCon](https://github.com/taichi-dev/taichicon): Taichi Developer Conferences
+- More to come...
 
-We use these channels to report bugs, discuss design, show off demos and send announcements on a daily basis:
+### Citations
 
-- [GitHub Issues](https://github.com/taichi-dev/taichi/issues)
-- [GitHub Discussions](https://github.com/taichi-dev/taichi/discussions)
-- [Twitter](https://twitter.com/taichigraphics)
-- [Taichi 中文论坛](https://forum.taichi.graphics/)
-- Slack & Wechat groups: please send us a message at <a href = "mailto:contact@taichi.graphics">contact@taichi.graphics</a> first, thanks!
+If you use Taichi in your research, please cite the corresponding papers:
 
-Should you spot any security issue, do not hesitate to report it by mailing to <a href = "mailto:security@taichi.graphics?subject = Taichi Security Problem">security@taichi.graphics</a>.
+- [(SIGGRAPH Asia 2019) Taichi: High-Performance Computation on Sparse Data Structures](https://yuanming.taichi.graphics/publication/2019-taichi/taichi-lang.pdf) [[Video]](https://youtu.be/wKw8LMF3Djo)|[[BibTex]](https://raw.githubusercontent.com/taichi-dev/taichi/master/misc/taichi_bibtex.txt)|[[Code]](https://github.com/taichi-dev/taichi)
+- [(ICLR 2020) DiffTaichi: Differentiable Programming for Physical Simulation](https://arxiv.org/abs/1910.00935) [[Video]](https://www.youtube.com/watch?v=Z1xvAZve9aE)|[[BibTex]|[[Code]](https://github.com/yuanming-hu/difftaichi)
+- [(SIGGRAPH 2021) QuanTaichi: A Compiler for Quantized Simulations](https://yuanming.taichi.graphics/publication/2021-quantaichi/quantaichi.pdf) [[Video]|[[BibTex]|[[Code]](https://github.com/taichi-dev/quantaichi)
 
 # Contributing
 
-If you would like to contribute to Taichi, please check out the [Contribution Guidelines](CONTRIBUTING.md).
-
-A huge thanks to all of our amazing contributors!
+Kudos to all of our amazing contributors! Taichi thrives through open-source. In that spirit, we welcome all kinds of contributions from the community. If you would like to participate, check out the [Contribution Guidelines](CONTRIBUTING.md) first.
 
 <a href="https://github.com/taichi-dev/taichi/graphs/contributors"><img src="https://raw.githubusercontent.com/taichi-dev/public_files/master/taichi/contributors_taichi-dev_taichi_12.png" width="800px"></a>
 
 *Contributor avatars are randomly shuffled.*
 
-# Resources
+# Community
 
-## Demos
+### Join our discussions
```

Comment:
Hmm, if you choose H3 just for visual reasons, I'd suggest to switch back to H2.